# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/loader/ModuleLoadRequest.cpp
# Commit: 360f8b5e412b
# Full Hash: 360f8b5e412b26defe0f6a01af1072ad7952ebfd
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-10-03 20:53 +0000
# Regressor Bug: 1968890
# File Overlap Count: 2
# Overlapping Files: js/loader/ModuleLoadRequest.cpp, js/loader/ModuleLoaderBase.cpp
# Description:
#   Bug 1968890 - Part 3: Call ModuleErrored in OnFetchFailed. r=jonco
# ==============================================================================

diff -r 1fe3698ab5d1 -r 360f8b5e412b js/loader/ModuleLoadRequest.cpp
--- a/js/loader/ModuleLoadRequest.cpp	Fri Oct 03 20:53:28 2025 +0000
+++ b/js/loader/ModuleLoadRequest.cpp	Fri Oct 03 20:53:28 2025 +0000
@@ -115,10 +115,6 @@
   MOZ_ASSERT(IsFetching() || IsPendingFetchingError());
 
   mModuleScript = mLoader->GetFetchedModule(ModuleMapKey(mURI, mModuleType));
-  if (IsErrored()) {
-    ModuleErrored();
-    return;
-  }
 }
 
 void ModuleLoadRequest::LoadFailed() {